README.md

```diff
@@ -10,6 +13,104 @@ The (evolving) protocol spec is rendered at [protocol.penumbra.zone][protocol].
 
 The (evolving) API documentation is rendered at [rustdoc.penumbra.zone][rustdoc].
 
+To participate in our test network, [keep reading below](#getting-started-on-the-test-network).
+
+For instructions on how to set up a node, [jump down and read on](#running-a-penumbra-node).
+
+## Getting started on the test network
+
+[Penumbra Labs][Penumbra] runs a test network for the latest version of the Penumbra protocol, and
+we would be delighted for you to try it out! Keep in mind that especially in the early stages of
+development, we expect many things to break. If you encounter bugs or wish for features to exist,
+we'd love for you to reach out to us on our [Discord server][Discord].
+
+### Building `pcli`
+
+To get started with the Penumbra test network, you will first need to download and build the
+Penumbra command line light wallet, `pcli`. This requires that you install a recent stable version
+of the Rust compiler, installation instructions for which you can find
+[here](https://www.rust-lang.org/learn/get-started).
+
+Once you have installed Rust and its build tool `cargo`, you can build the `pcli` light wallet by
+cloning this repository (`git clone https://github.com/penumbra-zone/penumbra`) and running from the
```

Comment:
Maybe this should be a code block so it's more easily copyable?

Reply:
Done.